app/assets/stylesheets/_mixins.scss

```diff
@@ -10,3 +10,18 @@
   #{$property}: #{$fallback};
   #{$property}: var(--#{$cssVariable}, #{$fallback});
 }
+
+/**
+*
+* Makes working with themeable CSS variables easier. Used when
+* a CSS variable requires !important as a postfix of the CSS property's
+* value where the CSS variable is being used.
+*
+* $property: the CSS property to theme.
+* $cssVariable: The CSS variable that will store the themed value.
+* $fallback: The value to use as a fallback but as well for the CSS variable default value.
+*/
+@mixin themeable-important($property, $cssVariable, $fallback) {
```

Comment:
Added this mixin to handle cases where `!important` is required but is not part of the value of the CSS variable.

Reply:
I like this